content/news/buzz/2022q4-roadmap/contents.lr

```diff
@@ -0,0 +1,75 @@
+* **Documentation** - Toga's documentation has lagged behind an acceptable standard for a long time,
+* **Planning** - Developing a clear plan for what will be in Toga 1.0
+* **Cross-platform hardware** - While it has always been *technically* possible to support the APIs needed to access device hardware, there hasn't been a cross-platform API for this work. In Q4, we hope to demonstrate at least 1 interesting piece of mobile device hardware.
+
+Although we won't get to Toga v1.0 in Q4, we *will* land a v0.3 final, after several years of maintaining dev pre-releases as the main way to use Toga.
+
+Web support
+~~~~~~~~~~~
+
+Toga has historically had support for the Web as a platform; however, that effort was put on hold because the approach we were taking wasn't proving viable. However, with the release of Python 3.11, WASM becomes an officially supported CPython platform. We've also seen the release of `PyScript <https://pyscript.net>`__, demonstrating that client-side Python in the browser is an extremely viable proposition. The time has come to resurrect BeeWare's support for the web as a deployment platform. This will involve getting Toga's web backend into a working state, and adding a web deployment backend for Briefcase.
+
+Longer term goals
+-----------------
+
+Looking into next year, the big items on the horizon are:
+
+Upstreaming
+~~~~~~~~~~~
+
+With binary modules now being supported on both iOS and Android, and the introduction of a `Tier 3 support level in PEP11 <https://peps.python.org/pep-0011/>`__,  the prospect of elevating iOS and Android to official supported platforms is seeming increasingly plausible. Our hope is to go to PyCon US next year in a position to make a concrete proposal to the CPython core team.
+
+Toga 1.0
+~~~~~~~~
+
+The broad goal for "Toga 1.0" could be considered "Tkinter, but with some additional bells to show off". We want to deliver a core widget set of the "essentials", plus some nice features to show off, like WebViews and camera or GPS integration. We hope to establish the feature list for Toga 1.0 in Q4; once that plan is in place, we'll have a better idea of when we will be able to deliver on that plan.
+
+A Forge for mobile packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As of today, all binary modules for mobile platforms are being maintained by the BeeWare team. This isn't a viable solution long term; we need to move to an environment where the community can manage binary packages for mobile platforms.
+
+Summary
+--------
+
+For the last 6 months, the focus of BeeWare has been on Briefcase and the support packages - ensuring that we have a feature complete, robust set of tools for deploying apps on any platform. In Q4, our focus is shifting to the tools we need to build apps *on* those platforms, and ensuring we have a stable foundation for future development. It will take time for those tools to reach maturity, but 
+
+---
+pub_date: 2022-10-04
```

Comment:
I would finish this sentence for you, but

Reply:
🤣  I really do have the attention span of a sparrow...